### PR TITLE
add GGIoU

### PR DESCRIPTION
**Pull request by eungaeun**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

**WARNING ⚠️** this PR is very large, summary may not cover all changes.

### 🌟 Summary
Introducing improvements and new features in image and model processing.

### 📊 Key Changes
- Added new images and model predictions for comprehensive test coverage.
- Updated labeling for frames, enhancing the model's accuracy and performance.
- Improved image handling, including rotation and adjustments, for better model training.

### 🎯 Purpose & Impact
- Enhances the model's ability to understand and predict based on a wider array of input data.
- Aims to improve accuracy, performance, and robustness of the model through more comprehensive test cases and corrected labels.
- Improves user experience by ensuring more reliable and precise outcomes from model predictions.

### CODE REVIEW

**Comment by glenn-jocher** [comment]
@eungaeun thanks for the suggestion on adding GIoU! 🌟 We're always looking at ways to improve and such feedback is invaluable. Currently, we're focused on the enhancements and features outlined in the PR summary, aiming to boost model performance and accuracy.

Should we explore adding GIoU or other loss functions in the future, we'll definitely keep the community posted. For now, you can implement it as a custom modification. For example:

```python
def giou_loss(preds, targets):
    # Implementation here
    return giou

# Use the custom loss function in your model's training loop
loss = giou_loss(predictions, targets)
```

Again, thank you for your input, and stay tuned for more updates!
